### ISSUE TITLE
Packages and modules are in different name spaces

### ISSUE THREAD

**Issue by andrewthetechie** [Narasimha1997/pyMigrate#2]
This is a cool idea but is fundamentally broken.

Try on this code:

```import yaml

with open('items.yaml') as f:
    
    data = yaml.load(f, Loader=yaml.FullLoader)
    print(data)
```

That simple example will fail because "import yaml" requires you to install the package pyyaml. 

Just a few more example packages that would be broken in this same way

bs4 
torch 
Pillow
sklearn
scikit-learn

**Comment by Narasimha1997** [comment]
@andrewthetechie 
Yes you're right. I was aware of this issue, but I forgot to specify this in README. 

Now I've updated the README so that the users who try to use this tool are aware this problem exists. 

I don't see any reliable way to do this, because of the fundamental nature of python's package management. The same is not true in Go or Node.js. I'm trying to see if there is any API in PyPi that allows us to atleast identify packages to some level of accuracy, if package name != Module name. 

I've updated the README to reflect this issue. If you have any solution in mind, or if you come across any possible solution in the future, please let me know in this thread.

**Comment by Narasimha1997** [comment]
@andrewthetechie 
This issue is satisfactorily fixed. 
The recent version can search in the module paths and get the Package name from Package info. If the package is not installed natively on the system, it will prompt the user to enter the package name. Each of the dependencies can be selected manually and new dependencies can be added as well. 

Checkout the latest version.

Thanks. Closing this as of now.

**Comment by andrewthetechie** [comment]
Very cool!
